packages/@aws-cdk/aws-cognito-identitypool-alpha/README.md

```diff
@@ -179,19 +180,19 @@ new IdentityPool(this, 'myidentitypool', {
   identityPoolName: 'myidentitypool',
   authenticationProviders: {
     amazon: {
-      appId: 'amzn1.application.12312k3j234j13rjiwuenf',
+      clientId: 'amzn1.application.12312k3j234j13rjiwuenf',
     },
     facebook: {
-      appId: '1234567890123',
+      clientId: '1234567890123',
```

Comment:
Is this better? A user will know that "I want to use facebook, I should find something called "appId""

In this new method, they have to find the non-existant "clientId"

Reply:
This was something I'd been debating after raising the PR, and after some thought I do agree that it's probably not worth the change for a _slightly_ cleaner backend. Will revert this and keep the old attribute names.